src/PrestaShopBundle/Resources/config/services/core/cldr.yml

```diff
@@ -1,8 +0,0 @@
-services:
-    _defaults:
-        public: true
-
-    prestashop.core.cldr.repository:
```

Comment:
We can't do that. It's a BC break if someone built a shop on PS 1.7.5 and started using this service. Do you wish to deprecate it ?

Reply:
It was introduced on October 4th, 2018 in this commit https://github.com/PrestaShop/PrestaShop/commit/99aec1594844fc6c94f69756a6296d8f615f4a7d.

I removed it due to @sarjon told that this service already exists in

`src/PrestaShopBundle/Resources/config/services/core/localization/cldr.yml` but now I see that it was introduced on October 10th, 2018 so @sarjon should remove hes  service and I should keep it :rofl: 

So this code is in 1.7.5 and maybe we should deprecate it only?

Reply:
OK I see it now.
You're right: we can delete it, but not here because I know at least one other PR that relies on one of these services.
Please use the service you want 😄 and I'll make a PR that deletes one of them and "migrate" all usages to the right one.

Reply:
Issue created: https://github.com/PrestaShop/PrestaShop/issues/12118